worldedit-core/src/main/java/com/boydti/fawe/beta/implementation/lighting/NMSRelighter.java

```diff
@@ -823,7 +814,8 @@ public void fixBlockLighting() {
         }
     }
 
-    public synchronized void flush() {
```

Comment:
For sake of consistency, flush() should be re-added and simply forward to close(). Might be worth adding a flushAsynce as well (similar to how EditSession works)